mobile-ui/src/main/java/com/playone/mobile/ui/injection/module/BrowseActivityModule.kt

```diff
@@ -15,6 +33,77 @@ import dagger.Provides
 @Module
 open class BrowseActivityModule {
 
+    @Provides
+    internal fun provideGetBufferoos(bufferooRepository: BufferooRepository,
+                                     threadExecutor: ThreadExecutor,
+                                     postExecutionThread: PostExecutionThread)
+            = GetBufferoos(bufferooRepository, threadExecutor, postExecutionThread)
+
+    @Provides
+    internal fun provideBufferooEntityMapper() = BufferooEntityMapper()
+
+    @Provides
+    internal fun provideBufferooMapper() = com.playone.mobile.presentation.mapper.BufferooMapper()
+
+    @Provides
+    internal fun provideDbBufferooMapper() = com.playone.mobile.cache.db.mapper.BufferooMapper()
+
+    @Provides
+    internal fun provideRemoteBufferooMapper()
+            = com.playone.mobile.remote.mapper.BufferooEntityMapper()
+
+    @Provides
+    internal fun provideDataBufferooMapper() = com.playone.mobile.data.mapper.BufferooMapper()
+
+    @Provides
+    internal fun provideUiBufferooMapper() = com.playone.mobile.ui.mapper.BufferooMapper()
+
+    @Provides
+    internal fun provideBufferooCache(factory: DbOpenHelper,
+                                      entityMapper: BufferooEntityMapper,
+                                      mapper: com.playone.mobile.cache.db.mapper.BufferooMapper,
+                                      helper: PreferencesHelper): BufferooCache
+            = BufferooCacheImpl(factory, entityMapper, mapper, helper)
+
+    @Provides
+    internal fun provideBufferooRemote(
+            service: BufferooService,
+            factory: com.playone.mobile.remote.mapper.BufferooEntityMapper): BufferooRemote
+            = BufferooRemoteImpl(service, factory)
+
+    @Provides
+    internal fun provideBufferooDataStoreFactory(bufferooCache: BufferooCache,
+                                                 bufferooCacheDataStore: BufferooCacheDataStore,
+                                                 bufferooRemoteDataStore: BufferooRemoteDataStore)
+            : BufferooDataStoreFactory
+            = BufferooDataStoreFactory(
+            bufferooCache,
```

Comment:
The format was crooked 😆.

```kotlin
BufferooDataStoreFactory(bufferooCache,
                         bufferooCacheDataStore,
                         bufferooRemoteDataStore)
```

The style will be as like this after we decided.

Reply:
Got trouble on align the parameters.........